package.json

```diff
@@ -4,11 +4,19 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/eslint/bin/eslint.js src/*/*.js"
```

Comment:
Only `eslint` is enough: when running npm scripts all the node_modules bin are in the PATH